test/OrchardCore.Tests/Modules/OrchardCore.Redis/RedisCacheWrapperTests.cs

```diff
@@ -0,0 +1,22 @@
+using OrchardCore.Redis.Services;
+using Microsoft.Extensions.Caching.Distributed;
+
+namespace OrchardCore.Tests.Modules.OrchardCore.Redis;
+
+public class RedisCacheWrapperTests
+{
+    [Fact]
+    public void RedisCacheWrapperMustNotDisposeInnerCache()
+    {
+        var redisCache = new Mock<IDistributedCache>();
+        var disposable = redisCache.As<IDisposable>();
+
+        object testObject = new RedisCacheWrapper(redisCache.Object);
```

Comment:
```suggestion
        var testObject = new RedisCacheWrapper(redisCache.Object);
```

Reply:
This must be an object explicitly, as `RedisCacheWrapper` currently doesn't implement `IDisposable`.